docs/001_develop/03_client-capabilities/005_grids/003_grid-pro/grid-pro_02_datasources.mdx

```diff
@@ -0,0 +1,197 @@
+---
+title: 'Datasources'
+sidebar_label: 'Datasources'
+id: grid-pro-datasources
+keywords: [genesis, foundation, ui, grid, grid-pro, ag, datasource, client-side, server-side]
+tags:
+  - connected
+  - datasource
+  - grid
+  - grid-pro
+  - ag
+  - client-side
+  - server-side
+pagination_prev: develop/client-capabilities/grids/grid-pro/grid-pro
+pagination_next: develop/client-capabilities/grids/grid-pro/grid-pro-cell-and-column
+---
+
+# Datasources
+
+## Overview
+
+The `@genesislcap/grid-pro` package provides a collection of grid-pro datasource components. These components are responsible for fetching data for the Genesis Grid Pro component.
+
+## `<grid-pro-client-side-datasource>`
+
+### Client-Side Features
+
+- **Sorting**: Order by any column, ASC or DESC.
+- **Filtering**:
```

Comment:
There is an include file you can import that would be good here to explain the filters https://github.com/genesiscommunitysuccess/docs/blob/new-doc-structure/docs/_includes/_groovy-filter-expressions.mdx